api/v1/controllers/RSVPController.js

```diff
@@ -0,0 +1,146 @@
+var bodyParser = require('body-parser');
+
+var services = require('../services');
+var middleware = require('../middleware');
+var requests = require('../requests');
+var roles = require('../utils/roles');
+var mail = require('../utils/mail');
+var User = require('../models/User');
+
+var router = require('express').Router();
+
+function _isAuthenticated (req) {
+    return req.auth && (req.user !== undefined);
+}
+
+function _removeFromList(rsvpCurrent, rsvpNew) {
+    if(rsvpCurrent.get('attendeeAttendance') !== 'NO' && rsvpNew.attendeeAttendance === 'NO')
+        return true;
+    return false;
+}
+
+function _addToList(rsvpCurrent, rsvpNew) {
+    if(rsvpCurrent.get('attendeeAttendance') === 'NO' && rsvpNew.attendeeAttendance !== 'NO')
```

Comment:
Yeah I've tested it out, I don't believe the require's in the index of the services use the js extension so it worked out fine.